### PR TITLE
rollback zookeeper to 3.6.3 and setup standaloneEnabled=false and add server.X by default to zoo.cfg, rollback maxResponseCacheSize, maxGetChildrenResponseCacheSize to default, add test_zookeeper.py which allow test ZK scale-out/scale-down all tests 0.15.0 PASSED

### PR DESCRIPTION
**Pull request by Slach**
- setup standaloneEnabled=false and add server.X by default to zoo.cfg, 
- add test_zookeeper.py which allow test ZK scale-out/scale-down 
- rollback zookeeper to 3.6.3 
- rollback maxResponseCacheSize, maxGetChildrenResponseCacheSize to default
- all tests 0.15.0 PASSED

Signed-off-by: Slach <bloodjazman@gmail.com>

Items PR complies to:
* [x] All commits in the PR are squashed. [More info](https://github.com/Altinity/clickhouse-operator/blob/master/CONTRIBUTING.md#how-to-make-pr) 
* [x] The PR is made into dedicated `next-release` branch, **not into** `master` branch<sup>1</sup>. [More info](https://github.com/Altinity/clickhouse-operator/blob/master/CONTRIBUTING.md#how-to-make-pr)
* [x] The PR is signed. [More info](https://github.com/Altinity/clickhouse-operator/blob/master/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

File: deploy/zookeeper/quick-start-persistent-volume/zookeeper-1-node.yaml [hunk 1]
```diff
@@ -92,11 +92,14 @@ spec:
       containers:
         - name: kubernetes-zookeeper
           imagePullPolicy: IfNotPresent
-          image: "docker.io/zookeeper:3.6.1"
+          image: "docker.io/zookeeper:3.7.0"
           resources:
             requests:
-              memory: "4Gi"
+              memory: "512M"
```
Comment (Slach) [inline deploy/zookeeper/quick-start-persistent-volume/zookeeper-1-node.yaml:98]: we already re-define requests \ limits on ACM side

File: deploy/zookeeper/quick-start-persistent-volume/zookeeper-1-node.yaml [hunk 2]
```diff
@@ -139,7 +142,9 @@ spec:
                 echo 'snapCount=3000000'
                 echo 'leaderServes=yes'
                 echo 'standaloneEnabled=true'
-                echo '4lw.commands.whitelist=stat, ruok, conf, isro, mntr'
+                echo '4lw.commands.whitelist=*'
+                echo 'maxResponseCacheSize=1200'
```
Comment (alex-zaitsev) [inline deploy/zookeeper/quick-start-persistent-volume/zookeeper-1-node.yaml:146]: Do we have enough RAM for that?

Comment (Slach) [reply deploy/zookeeper/quick-start-persistent-volume/zookeeper-1-node.yaml:146]: according to internal grafana, currently, we use 20-30% of available memory on ZK

**Comment by alex-zaitsev** [comment]
@Slach , why do we need to upgrade to 3.7.0? The release is pretty new. Have you made any research on that?
For example, there are 12 issues that affect 3.7.0 in Jira and only 9 for 3.6.3:
https://issues.apache.org/jira/issues/?jql=project%20%3D%20ZOOKEEPER%20AND%20resolution%20%3D%20Unresolved%20AND%20affectedVersion%20%3D%203.7.0%20ORDER%20BY%20priority%20DESC%2C%20updated%20DESC

**Comment by Slach** [comment]
ok. let's wait more releases 3.7.x

File: deploy/zookeeper/quick-start-persistent-volume/zookeeper-1-node.yaml [hunk 3]
```diff
@@ -139,7 +142,9 @@ spec:
                 echo 'snapCount=3000000'
                 echo 'leaderServes=yes'
                 echo 'standaloneEnabled=true'
```
Comment (alex-zaitsev) [inline deploy/zookeeper/quick-start-persistent-volume/zookeeper-1-node.yaml:144]: standaloneEnabled=true, should be false. I also do not see the server.1 change here.

File: deploy/zookeeper/quick-start-persistent-volume/zookeeper-1-node.yaml [hunk 4]
```diff
@@ -92,11 +92,14 @@ spec:
       containers:
         - name: kubernetes-zookeeper
           imagePullPolicy: IfNotPresent
-          image: "docker.io/zookeeper:3.6.1"
+          image: "docker.io/zookeeper:3.7.0"
```
Comment (alex-zaitsev) [inline deploy/zookeeper/quick-start-persistent-volume/zookeeper-1-node.yaml:95]: Should be 3.6.3

File: deploy/zookeeper/quick-start-persistent-volume/zookeeper-3-nodes.yaml [hunk 5]
```diff
@@ -90,11 +90,14 @@ spec:
       containers:
         - name: kubernetes-zookeeper
           imagePullPolicy: IfNotPresent
-          image: "docker.io/zookeeper:3.6.1"
+          image: "docker.io/zookeeper:3.7.0"
```
Comment (alex-zaitsev) [inline deploy/zookeeper/quick-start-persistent-volume/zookeeper-3-nodes.yaml:93]: Should be 3.6.3

**Review by alex-zaitsev — CHANGES_REQUESTED** [review]

File: deploy/zookeeper/quick-start-persistent-volume/zookeeper-1-node.yaml (hunk 3 above)
Comment (Slach) [reply deploy/zookeeper/quick-start-persistent-volume/zookeeper-1-node.yaml]: fixed on https://github.com/Altinity/acm/commit/9455cf6e87a95de7c0ff36c221d3fe88988c1f79

File: deploy/zookeeper/quick-start-persistent-volume/zookeeper-3-nodes.yaml (hunk 5 above)
Comment (Slach) [reply deploy/zookeeper/quick-start-persistent-volume/zookeeper-3-nodes.yaml]: fixed on https://github.com/Altinity/acm/commit/9455cf6e87a95de7c0ff36c221d3fe88988c1f79

**Review by alex-zaitsev — APPROVED** [review]

**Pull request merged**